base/commands/migration/migration_start.go

```diff
@@ -45,14 +53,34 @@ Selected data structures in the source cluster will be migrated to the target cl
 		}
 	}
 	ec.PrintlnUnnecessary("")
-	sts := NewStages(makeMigrationID(), ec.GetStringArg(argDMTConfig))
+	mID := MakeMigrationID()
+	sts := NewStartStages(ec.Logger(), mID, ec.GetStringArg(argDMTConfig))
 	sp := stage.NewFixedProvider(sts.Build(ctx, ec)...)
 	if _, err := stage.Execute(ctx, ec, any(nil), sp); err != nil {
 		return err
 	}
+	mStages, err := createMigrationStages(ctx, ec, ci, mID)
+	if err != nil {
+		return err
+	}
+	mp := stage.NewFixedProvider(mStages...)
+	_, err = stage.Execute(ctx, ec, any(nil), mp)
+	err2 := finalizeMigration(ctx, ec, ci, mID, ec.Props().GetString(flagOutputDir))
+	if err2 != nil {
+		return fmt.Errorf("finalizing migration: %w", err2)
+	}
+	if err != nil {
+		return err
+	}
 	ec.PrintlnUnnecessary("")
 	ec.PrintlnUnnecessary("OK Migration completed successfully.")
-	return nil
+	return ec.AddOutputRows(ctx, output.Row{
+		output.Column{
+			Name:  "Migration ID",
+			Type:  serialization.TypeString,
+			Value: mID,
+		},
+	})
```

Comment:
what does this do? I don't see a table shown after the migration is done in your demo video on slack.

Reply:
I just added it later, that is why.